.github/pull-request-template.md

```diff
@@ -0,0 +1,15 @@
+## Summary
+<!-- A brief summary of the PR -->
+
+## Checklist
+<!-- [ ] = no, [x] = yes -->
+
+ - [ ] I have searched for similar/duplicate PR's, and there are none.
+ - [ ] This is a **documentation update** (main code base unchanged).
+ - [ ] This is a **source code update** (main code base changed).
+   - [ ] I have created unit tests for the modified code base.
+   - [ ] All tests pass (if not please elaborate).
+   - [ ] This is a breaking change (requires major version update).
+
+## Related issues
+<!-- Any issues this PR addresses -->
```

Comment:
Again, just another simple change to be formatted a specific way that I like (same way I have done my previous PRs). I should also be able to fix this as well.

```suggestion
<!-- Any issues this PR addresses. When listing the issues, please follow the example format below, making a list with the issues. -->
<!-- - Closes #issue-num  -->
```